public/css/main.css

```diff
@@ -18,5 +17,5 @@ main {
 }
 
 form {
-  margin:0 auto;
+  margin: 0 auto;
```

Comment:
Be consistent with spaces before `{` and after `:`.. Always do it or never do it - preferably the first, but either happens in the wild.